Source/WebCore/rendering/RenderBlockFlow.h

```diff
@@ -551,6 +551,8 @@ class RenderBlockFlow : public RenderBlock {
     // FIXME: This is temporary until after we remove the forced "line layout codepath" invalidation.
     std::optional<LayoutUnit> m_previousModernLineLayoutContentBoxLogicalHeight;
 
+    Vector<WeakPtr<RenderLayerModelObject>> m_descendantsNeedingTransformUpdate;
```

Comment:
This is bloating RenderBlockFlow (of which there are potentially thousands on a web page) with a data member whose lifetime is transient; this member is only used during layout, so it should state on the stack or some per-layout state object.

Reply:
Moved to LocalFrameViewLayoutContext. Not sure this is ideal but we can't use the RenderLayoutState stack since we would need the containing block's state not the current state (although maybe its ok to add a method that searches through the stack for the parent's state).